### DISCUSSION TITLE
A question about content_ui

### DISCUSSION THREAD

**Discussion by Ya-hui** [wxt-dev/wxt#1071] [Q&A]
A question: I have N pages that need to be injected with content ui. Should I create a content for each page or just create one content and then use if in the code to determine whether to display it? Creating a content for each page will make the package very large.

**Comment by aklinker1** [comment]
Both work. If package size is your main concern, do whatever results in a smaller size.

**Reply by Ya-hui** [reply]
This is your first time developing a Chrome extension. Which method do you think is more suitable? If you only write a content_ui page and make judgments at the code level, how do you import the CSS style? Is it dynamic import?

**Reply by aklinker1** [reply]
I prefer having a single content script per website. 

As for CSS, just import your CSS files at the top of your content script:

```ts
import "~/assets/global.css";

export default defineContentScript(...);
```

**Reply by Ya-hui** [reply]
Yes, I also like that each website has a content script, which is clearer. I use Vue and some third-party libraries such as vue.js, md5.js, crypto.js. This will make each content script package these third-party libraries, and the package size will be very large. How can I change it to static import instead of packaging them all into the content script?
